templates/web/camden/before_wrapper.html

```diff
@@ -0,0 +1,6 @@
+[% IF c.config.BASE_URL == "https://www.fixmystreet.com" OR c.config.BASE_URL == 'https://camden.staging.fixmystreet.com' %]
```

Comment:
`c.config.BASE_URL` on staging is actually `https://staging.fixmystreet.com`.

Have fixed in the `camden-reviewed` branch (https://github.com/mysociety/fixmystreet/commit/283cbb31c44048deef84a1d37c38d565db73c96e) and redeployed to staging. Seems to be working now.